lib/reducers/wallets.js

```diff
@@ -0,0 +1,23 @@
+import { USER_SELECT_WALLET, USER_SELECT_ACCOUNT } from '../constants';
+
+const initialState = {
+  selectedWallet: 'primary',
+  selectedAccount: 'default'
+};
+
+export default function walletsReducer(state = initialState, action) {
+  const { type, payload = {} } = action;
+  let newState = { ...state };
+
+  const { walletId, accountId } = payload;
+  switch (type) {
+    case USER_SELECT_WALLET:
+      newState.selectedWallet = walletId;
```

Comment:
This change alters the location of `selectedWallet` in the store, going from `PLUGIN_NAMESPACE.selectedWallet` to `wallets.selectedWallet`
```
case USER_SELECT_WALLET:
  newState = safeSet(newState, `${PLUGIN_NAMESPACE}.selectedWallet`, walletId);
  return newState;
```

Reply:
The shape should actually still be retained. The new reducer structure lets plugins say what their reducer will be and still puts it under `plugins.[NAMESPACE]`. You can see this in the screenshot from the redux devtools below. I've confirmed also that the compatibility between plugins still works as the footer address plugin which relies on selectedWallet still works.

<img width="616" alt="screen shot 2018-07-09 at 5 53 28 pm" src="https://user-images.githubusercontent.com/4344978/42483206-0d0e8802-83a1-11e8-96b2-3853d1c0c9b3.png">

Reply:
This is where it gets added to a plugins store: https://github.com/bpanel-org/bpanel/pull/50/files#diff-847252183981a28fa3823a7a41176071R29

Reply:
Cool! 👍